addons/website/views/snippets/snippets.xml

```diff
@@ -1952,6 +1952,14 @@
             <we-input string="Small" class="o_we_sublevel_1" data-customize-website-variable="" data-variable="input-border-radius-sm" data-unit="px" data-save-unit="rem"/>
             <we-input string="Large" class="o_we_sublevel_1" data-customize-website-variable="" data-variable="input-border-radius-lg" data-unit="px" data-save-unit="rem"/>
         </we-collapse>
+        <we-row string="Background">
+            <we-colorpicker title="Background"
+                data-no-transparency="true"
+                data-customize-website-variable="input-background-color"
+                data-variable="input-background-color"
+                data-color="input-background-color"
+                data-use-css-color="true"/>
```

Comment:
Maybe it's this param that you can remove once you made the value a color instead of a "website value".

Reply:
Nope, the default color is still "transparent"